src/about/readme.md

```diff
@@ -25,6 +25,12 @@ Mr.Hope 是一个非常和气的人。
 
 - [副室长简历](/file/林传舜简历.pdf)
 
+## 成员
+
+李欣桐，软件工程 2019级本科生。
+
+年轻帅气有活力,聪明机智有节操

```

Comment:
Line 30: Type space between chinese character and number, and chinese character and english letter.
Line 32 : Do use `，` instead of `,` in chinese lines.

So, I would prefer:

```md
## 成员

### 李欣桐

软件工程 2019 级本科生

年轻帅气有活力，聪明机智有节操。
```